src/Joomlatools/Console/Command/Site/Configure.php

```diff
@@ -135,20 +135,21 @@ protected function _configureJoomlaCMS()
 
         $contents = file_get_contents($source);
         $replace  = function($name, $value, &$contents) {
-            $pattern = sprintf('#\$%s\s*=\s*(["\']).*?(?<!\\\\)\1#', $name);
+            $pattern = sprintf('#\$%s\s*=\s*(["\']?).*?\1(?=[;\1])#', $name);
             $match   = preg_match($pattern, $contents);
+            $value   = ctype_digit($value) ? $value : "'" . str_replace("'", "\\'", $value) . "'";
```

Comment:
@msaladna I suggest we use [is_numeric()](http://php.net/manual/en/function.is-numeric.php) instead of [ctype_digit()](http://php.net/manual/en/function.ctype-digit.php) here. `ctype_digit` will return `false` for integers between -128 and 255, because they get interpreted as the ASCII value of a single character.

Example from the docs:

```php
ctype_digit('42'); // true
ctype_digit(42); // false (42 is ASCII code for the * char)
```

Reply:
You're right. Cue the "shit!" on my end for not testing that data type.